### PR TITLE
[CodingStyle] Remove previous node attribute on TernaryConditionVariableAssignmentRector

### PR DESCRIPTION
**Pull request by samsonasik**
Replace `previous` with `assigned_to` attribute on the ternary.

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Awesome :+1:

File: packages/NodeTypeResolver/Node/AttributeKey.php [hunk 1]
```diff
@@ -192,4 +192,9 @@ final class AttributeKey
      * @var string
      */
     public const NEWLINED_ARRAY_PRINT = 'newlined_array_print';
+
+    /**
+     * @var string
+     */
+    public const ASSIGNED_TO = 'assigned_to';
```
Comment (samsonasik) [inline packages/NodeTypeResolver/Node/AttributeKey.php:199]: @TomasVotruba I just re-check that `assigned_to` attribute came from `symplify/phpstan-rules`, which is a `require-dev`, on scoped build, it is not included, so the `AssignedToPropertyNodeVisitor` need to be copied to rector-src

https://github.com/symplify/phpstan-rules/blob/main/src/NodeVisitor/AssignedToPropertyNodeVisitor.php#L15

Comment (samsonasik) [reply packages/NodeTypeResolver/Node/AttributeKey.php:199]: https://github.com/rectorphp/rector-src/pull/3694

**Pull request merged**